src/main/java/org/folio/edge/sip2/repositories/PatronRepository.java

```diff
@@ -159,6 +159,16 @@ private Future<PatronInformationResponse> validPatron(String userId, Personal pe
     final Future<PatronInformationResponseBuilder> manualBlocksFuture = feeFinesRepository
         .getManualBlocksByUserId(userId, sessionData)
         .map(blocks -> buildPatronStatus(blocks, builder));
+    // add fine count
+    final Future<PatronInformationResponseBuilder> accountFuture = feeFinesRepository
+        .getAccountDataByUserId(userId, sessionData)
+        .map(accounts -> populateFinesCount(accounts,builder));
```

Comment:
Would add a space between the params: `accounts,builder` should be `accounts, builder`.

Reply:
Added